docs/get-started/using-with-map.md

```diff
@@ -49,17 +49,17 @@ Existing free vector tile services:
 - [CARTO free basemaps](https://carto.com/basemaps) for non commercial applications:
 
 
-| POSITRON | DARK MATTER | VOYAGER  |
+
+| NAME | PREVIEW | LINK  |
```

Comment:
I think it might be good to be a little more explicit that this JSON URL is what should be passed to `mapStyle`. I could see someone clicking the link, seeing JSON, and not knowing what to do with it.

Reply:
That's explained in the previous paragraph.

Reply:
There's nothing in the previous paragraph that references a JSON URL. Most people who are new to Mapbox GL JS have no idea what a Mapbox GL style url should look like. Especially because most Mapbox GL JS users have only seen URLs that look like

```
mapbox://styles/mapbox/streets-v11
```

Reply:
I think one sentence like

> ... using the `mapStyle` prop. Note that for non-Mapbox services, this URL must link to a JSON file that conforms to the [Mapbox Style Specification](https://docs.mapbox.com/mapbox-gl-js/style-spec/).

is necessary.